pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -269,17 +269,20 @@ func (r *ReconcileVirtualMachineImport) createVM(provider provider.Provider, ins
 	}
 
 	// Update condition to creating VM:
-	cond := conditions.NewProcessingCondition(string(v2vv1alpha1.CreatingTargetVM), "Creating virtual machine")
-	if err = r.upsertStatusCondition(instanceNamespacedName, cond); err != nil {
+	processingCond := conditions.NewProcessingCondition(string(v2vv1alpha1.CreatingTargetVM), "Creating virtual machine", corev1.ConditionTrue)
+	if err = r.upsertStatusConditions(instanceNamespacedName, processingCond); err != nil {
 		return "", err
 	}
 
 	// Create kubevirt VM from source VM:
 	reqLogger.Info("Creating a new VM", "VM.Namespace", vmSpec.Namespace, "VM.Name", vmSpec.Name)
 	if err = r.client.Create(context.TODO(), vmSpec); err != nil {
 		// Update condition to failed state:
-		cond = conditions.NewSucceededCondition(string(v2vv1alpha1.VMCreationFailed), fmt.Sprintf("Error while creating virtual machine: %s", err), corev1.ConditionFalse)
-		err = r.upsertStatusCondition(instanceNamespacedName, cond)
+		succeededCond := conditions.NewSucceededCondition(string(v2vv1alpha1.VMCreationFailed), fmt.Sprintf("Error while creating virtual machine: %s", err), corev1.ConditionFalse)
+		processingCond.Status = corev1.ConditionFalse
```

Comment:
Could you set a reason for the new condition status  ? So the user will know that the processing is 'False' since it was completed successfully ?